package.json

```diff
@@ -0,0 +1,33 @@
+{
+    "name": "eslint-config-fluid",
+    "version": "0.1.0",
+    "description": "fluid-project eslint configuration",
+    "main": "index.js",
+    "engines": {
+        "node": ">=4.0.0"
+    },
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/fluid-project/eslint-config-fluid.git"
+    },
+    "keywords": [
+        "eslint",
+        "eslintconfig",
+        "fluid",
+        "fluid-project"
+    ],
+    "author": "Fluid Project",
+    "license": "BSD-3-Clause",
+    "bugs": {
+        "url": "https://issues.fluidproject.org/projects/FLUID/issues"
+    },
+    "homepage": "http://fluidproject.org",
+    "peerDependencies": {
+        "fluid-eslint": ">=2.0.0"
```

Comment:
> The reason for our fork is that the processing of .eslintignore is broken in the main line edition.

Is that still true in latest versions, and are we tracking it in any way?

Also i'll remove the peerDependency. Seems if it's deprecated anyways, it shouldn't be an issue to drop it.